app/controllers/comments_controller.rb

```diff
@@ -9,11 +9,11 @@ def create
     @comment.user_id = current_user.id
     @comment.post_id = params[:post_id]
 
```

Comment:
I get an error when I try to add a comment on a post. Kindly fix the error.
![comment](https://user-images.githubusercontent.com/35504149/156799892-6c212ce6-6dfa-4359-97c8-95e82461050b.png)